frontend/src/scenes/surveys/CONTRIBUTING.md

```diff
@@ -0,0 +1,86 @@
+# Surveys
+
+## How to test changes
+
+### Changes on BE/FE only (i.e. posthog/posthog repo)
+
+You don't need to do anything special.
+
+If you're making changes to the frontend/backend only (i.e. this repo), you can [run the app locally](https://posthog.com/handbook/engineering/developing-locally) and test as expected.
+
+Be sure to write tests for your changes. Especially for changes that deal with logic changes (i.e. [`surveyLogic.ts`](https://github.com/PostHog/posthog/blob/master/frontend/src/scenes/surveys/surveyLogic.ts) or [`surveysLogic.ts`](https://github.com/PostHog/posthog/blob/master/frontend/src/scenes/surveys/surveysLogic.ts)).
```

Comment:
add a  perma link otherwise this is gonna be broken if the file is moved/renamed etc

Reply:
fixed